assets/js/modules/analytics/components/module/ModuleAcquisitionChannelsWidget/index.js

```diff
@@ -0,0 +1,208 @@
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, _n, _x, sprintf } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { DATE_RANGE_OFFSET, MODULES_ANALYTICS, STORE_NAME } from '../../../datastore/constants';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import PreviewTable from '../../../../../components/PreviewTable';
+import SourceLink from '../../../../../components/SourceLink';
+import { isZeroReport } from '../../../util';
+import TableOverflowContainer from '../../../../../components/TableOverflowContainer';
+import ReportTable from '../../../../../components/ReportTable';
+import Header from './Header';
+import PieChart from './PieChart';
+import { numFmt } from '../../../../../util';
+import MiniChart from '../../../../../components/MiniChart';
+import { Cell, Grid, Row } from '../../../../../material-components';
+import PropTypes from 'prop-types';
+
+const { useSelect } = Data;
+
+export default function ModuleAcquisitionChannelsWidget( { Widget, WidgetReportZero, WidgetReportError } ) {
+	const reportType = 'trafficsources-overview';
+
+	const {
+		hasFinishedResolution,
+		dateRangeNumberOfDays,
+		report,
+		error,
+		url,
+	} = useSelect( ( select ) => {
+		const dates = select( CORE_USER ).getDateRangeDates( { offsetDays: DATE_RANGE_OFFSET } );
+		const reportArgs = {
+			...dates,
+			dimensions: 'ga:channelGrouping',
+			metrics: [
+				{
+					expression: 'ga:sessions',
+					alias: 'Sessions',
+				},
+				{
+					expression: 'ga:users',
+					alias: 'Users',
+				},
+				{
+					expression: 'ga:newUsers',
+					alias: 'New Users',
+				},
+			],
+			orderby: [
+				{
+					fieldName: 'ga:users',
+					sortOrder: 'DESCENDING',
+				},
+			],
+			limit: 10,
+		};
+
+		return {
+			dateRangeNumberOfDays: select( CORE_USER ).getDateRangeNumberOfDays(),
+			error: select( STORE_NAME ).getErrorForSelector( 'getReport', [ reportArgs ] ),
+			hasFinishedResolution: select( STORE_NAME ).hasFinishedResolution( 'getReport', [ reportArgs ] ),
+			report: select( STORE_NAME ).getReport( reportArgs ),
+			url: select( MODULES_ANALYTICS ).getServiceReportURL( reportType ),
+		};
+	} );
+
+	if ( ! hasFinishedResolution ) {
+		return (
+			<Widget Header={ Header } noPadding>
```

Comment:
We need to remove `noPadding` here because we need padding when using placeholders.
```suggestion
			<Widget Header={ Header }>
```

Reply:
Updated in https://github.com/google/site-kit-wp/pull/3035/commits/262cfaac31149a2113b3e63f3000f9396ee0fc2b